src/components/ContinuationIn/ManualBusinessInfo.vue

```diff
@@ -216,6 +216,18 @@
           </v-col>
         </v-row>
 
+        <!-- Upload Affidavit -->
+        <UploadAffidavit
+          :business="business"
+          @updateBusiness="updateBusiness"
+        />
+
+        <!-- empty column -->
+        <v-col
+          cols="12"
+          sm="3"
+        />
```

Comment:
You shouldn't have a v-col that is not wrapped in a v-row.

If you want whitespace above the message box, add class "mt-6" to the message box row element.

Reply:
Removed, looks good now:
![image](https://github.com/bcgov/business-create-ui/assets/122495122/40ab9917-11e1-4266-acc6-b141592314c6)